README

```diff
@@ -1,16 +1,25 @@
-UCX is a communication library implementing high-performance messaging for MPI/PGAS frameworks
+UCX is a communication library implementing a high-performance messaging
+layer for MPI, PGAS, and RPC frameworks.
 
-### Quick start
+### Running internal unit tests
 ```sh
-$ export PATH=$PATH:$MPI_HOME/bin  
+$ ./autogen.sh
+$ ./contrib/configure-devel
+$ make
+$ make -C test/gtest test
+```
+
+### Running UCX performance test
```

Comment:
need example how to run perftest w/o mpi as well (ssh)

Reply:
+1 majority compiles without MPI. I would suggest to remove this MPI example all together. it is misleading !

Reply:
maybe better to mark it as "optional" or for convenience of mpi users